src/Microsoft.DocAsCode.Build.SchemaDriven/ApplyOverwriteUtility.cs

```diff
@@ -12,48 +12,71 @@ namespace Microsoft.DocAsCode.Build.SchemaDriven
     using Microsoft.DocAsCode.DataContracts.Common;
     using Microsoft.DocAsCode.Plugins;
 
-    public class ApplyOverwriteUtility
+    public class ApplyOverwriteHelper
```

Comment:
naming: `Helper`, `Utility`, as convention, sounds like static classes?

Reply:
what about OverwriteApplier?

---
In reply to: [165292115](https://github.com/dotnet/docfx/pull/2398#discussion_r165292115) [](ancestors = 165292115)